core/src/main/scala/org/apache/spark/ui/storage/ToolTips.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ui.storage
+
+private[ui] object ToolTips {
+  val ID =
+    "ID of RDD"
+
+  val RDD_NAME =
+    "Name of persisted RDD"
+
+  val STORAGE_LEVEL =
+    "StorageLevel displays where RDD is stored, " +
+      "Format of RDD (Serialized or De-serialized) and" +
+      "Replication factor of the RDD"
+
+  val CACHED_PARTITIONS =
+    "Number of partitions cached"
+
+  val FRACTION_CACHED =
+    "Fraction of RDD cached"
```

Comment:
Interesting, I actually dont' know whether is the fraction of partitions or data cached. That's not quite the same. Would be good to figure it out and clarify here